CMFA_ESR.adoc

```diff
@@ -42,7 +42,7 @@ This section needs to be reviewed
 ====
 The {itc-longname} iTC has been requested by vendors to develop an Essential Security Requirements (ESR) for Continuous Multifactor Authentication (CMFA) products. The CMFA ESR version 1.0 contains material generated by the members of the {itc-shortname}, for a PP-Module to be reviewed by the membership of the {itc-shortname} and endorsed in 2021.
 
-CMFA is an implicit process of validating the user periodically to grant access to the legitimate user and to deny access to impostors based on captured biometric characteristics and built-in sensors of the computer device.
+CMFA is an implicit process of validating the user periodically to grant access to the legitimate user and to deny access to impostors based on captured biometric characteristics and built-in sensors of the device (or connected devices).
```

Comment:
Suggest making the parenthetical part of the sentence and making this _and_ not _or_, to not imply one or the other.
```suggestion
CMFA is an implicit process of validating the user periodically to grant access to the legitimate user and to deny access to impostors based on captured biometric characteristics and built-in sensors of the device and connected devices.
```